packages/terra-application-utility/package.json

```diff
@@ -21,16 +21,19 @@
     "url": "https://github.com/cerner/terra-framework/issues"
   },
   "homepage": "https://github.com/cerner/terra-framework#readme",
+  "devDependencies": {
```

Comment:
We can drop these devDependencies. Terra-base should already be provided by terra-base and then we want terra-base to manager the react-intl version.

Reply:
Unfortunately since terra-base is a peer dependency npm does not install those and therefore causes errors in the tests

Reply:
Ah you are right about terra-base. Whoops! I was thinking since terra-dev-site installs this as well it would be handled.

Reply:
i'm curious if tests pass when we replace [`IntlProvider`](https://github.com/cerner/terra-framework/blob/master/packages/terra-application-utility/tests/jest/UtilityButton.test.jsx#L2) with a terra-base component - this way we can remove `react-intl` as a dev dep

Reply:
I don't think we export that - https://github.com/cerner/terra-core/blob/master/packages/terra-base/src/Base.jsx

Reply:
booooooo

Reply:
wait i'm confused, aren't we exporting it here: https://github.com/cerner/terra-core/blob/master/packages/terra-base/src/Base.jsx#L108

Reply:
Nope that's the base component which features the render, then the next line is where we `export injectIntl`